Competition2017/src/competition/subsystems/drive/commands/DriveForDistanceCommand.java

```diff
@@ -71,7 +80,21 @@ public void initialize() {
     public void execute() {
         double power = travelManager.calculate(targetDistance, driveSubsystem.getDistance());
         
-        driveSubsystem.tankDrivePowerMode(power, power);
+        double leftPower = power - calculateHeadingPower();
+        double rightPower = power + calculateHeadingPower();
+        
+        driveSubsystem.tankDrivePowerMode(leftPower, rightPower);
+    }
+    
+    public double calculateHeadingPower() {
+
+        double errorInDegrees = driveSubsystem.imu.getYaw().difference(targetHeading);
+        double normalizedError = errorInDegrees / 180;
+        double rotationalPower = headingDrivePid.calculate(0, normalizedError);
+
+        rotationalPower *= -1;
```

Comment:
Instead of negating it here, why not swap the operands in the `difference` call?